### PR TITLE
[sanity test] enable sanity check against 201811 images

### PR DESCRIPTION
**Pull request by yxieca**
Summary:
enable sanity check against 201811 images

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?
To enable sanity check when testing upgrade path from 201811 to an newer branch.

#### How did you do it?
- This change is needed for upgrade path test when source image is from 201811 branch.

Signed-off-by: Ying Xie <ying.xie@microsoft.com>

#### How did you verify/test it?
Run test_pretest.py::test_update_testbed_metadata, without the change, sanity check fails. With the change, test passes.

### CODE REVIEW

File: tests/common/devices.py [hunk 1]
```diff
@@ -1686,6 +1692,10 @@ def run_redis_cli_cmd(self, redis_cmd):
     def get_ip_route_info(self, dstip):
         return self.sonichost.get_ip_route_info(dstip, self.cli_ns_option)
 
+    @property
+    def os_version(self):
+        return self.sonichost.os_version
```
Comment (wangxin) [inline tests/common/devices.py:1697]: Why need to define this property? The `duthost` is an instance of `MultiAsicSonicHost`. Accessing `duthost.os_version` will eventually get `SonicHost.os_version`.
```
(Pdb) duthost
<tests.common.devices.MultiAsicSonicHost object at 0x7f7d667fcbd0>
(Pdb) duthost.os_version
u'master.606-3acf7006'
```

Comment (yxieca) [reply tests/common/devices.py:1697]: This is from AsicHost. AsicHost didn't define this attribute causes access exception.

Comment (wangxin) [reply tests/common/devices.py:1697]: But this SonicAsic.os_version property is not accessed in this PR's code change, right?

Comment (yxieca) [reply tests/common/devices.py:1697]: It is accessed by checks.py. The multi-asic refactoring causes the show_interface() being called from asichost level.

Comment (wangxin) [reply tests/common/devices.py:1697]: Ah, I see, the `dut` argument is an instance of duthost.asics. I was confused by its name.

**Review by wangxin — APPROVED** [review]

**Pull request merged**